set_bioinforesponsible.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+""" Calls up the genologics LIMS directly in order to more quickly
+    set a bioinformatics responsible. Script can easily be altered
+    to be used to set other values."""
+    
+from genologics.lims import Lims
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+def namesetter(PID):
+
+    lims = Lims(BASEURI, USERNAME, PASSWORD)
+    lims.check_version()
+    #Find LIMS entry with same PID
+    allProjects = lims.get_projects()
+    for proj in allProjects:
+        if proj.id == PID:
+            limsproject = proj.name
+            break
+    #Error handling
+    if not 'limsproject' in locals():
+        print "{} not available in LIMS.".format(PID)
+        return None
+    
+    #Enter project summary process
+    stepname=['Project Summary 1.3']
+    process=lims.get_processes(type=stepname, projectname=limsproject)
+    #Error handling
+    if process == []:
+        print "{} for {} is not available in LIMS.".format(stepname, limsproject)
+        return None
+
+    loop = True
+    while loop:
+        if "Bioinfo responsible" in process[0].udf:
+            response = process[0].udf["Bioinfo responsible"]
+        else:
+            response = "Unassigned"
+        print "Existing Bioinfo responsible for project {} aka {}: {}".format(limsproject, PID, response.encode('utf-8'))
+        newname = raw_input("Enter name of new Bioinfo responsible, or C to cancel: ")
+        if newname != 'c' and newname != 'C':
+            confirmation = raw_input("Project {} aka {} will have {} as new Bioinfo responsible, is this correct (Y/N)? ".format(limsproject, PID, newname))
+            if confirmation == 'Y' or confirmation == 'y':
+                try:
+                    process[0].udf["Bioinfo responsible"] = newname
```

Comment:
bioinformaticians are selected from a drop down window, what happens with this script if I enter a new name (e.g., "pippo")? It adds it as boiinfo responsible I assume. Would be much safer to check if the name entered is one of the allowed ones, and only in that case enter it.